code/lib/core-server/src/utils/new-story-templates/typescript.ts

```diff
@@ -0,0 +1,33 @@
+import dedent from 'ts-dedent';
+
+export function getTypeScriptTemplateForNewStoryFile(data: {
+  /** The components file name without the extension */
+  basename: string;
+  componentExportName: string;
+  default: boolean;
+  /** The framework package name, e.g. @storybook/nextjs */
+  frameworkPackageName: string;
+  /** The exported name of the default story */
+  exportedStoryName: string;
+}) {
+  const importName = data.default ? 'Component' : data.componentExportName;
```

Comment:
I think this should be `data.componentExportName` either way.

Reply:
`componentExportName` for a default export would be `default`. And since `default` is a reserved keyword, I would suggest to use a static string instead. It doesn't have to be `Component`, though.